maintenance/rebuildLocalisationCache.php

```diff
@@ -151,6 +158,28 @@ function ( $item ) use ( $firstCodes ) {
 	}
 
 	/**
+	 * Helper function to include all localisation files to localisation cache rebuild process.
+	 * It looks for i18n and alias files from whole code directory and registers them in global array.
+	 */
+	private function includeAllMessagesFiles() {
+		global $wgExtensionMessagesFiles;
```

Comment:
It is perfectly fine to use `global $wgExtensionMessagesFiles, $IP;` which we normally do everywhere else in the codebase.